strings/naiveStringSearch.go

```diff
@@ -16,7 +16,8 @@ import (
 
 func naivePatternSearch(text string, pattern string) []int {
 	var positions []int
-	for i := 0; i < len(text)-len(pattern); i++ {
+	// added the "=" sign to the condition in for loop i to include the last value which return 18
```

Comment:
We don't need this comment. Correct the algorithm is enough.

Reply:
Correct the algorithm.

Reply:
I mean we should remove this comment out of codebase. Because it doesn't help to provide any information related to the rest of the algorithm implementation.

Reply:
Correct the algorithm